README.md

```diff
@@ -41,7 +41,7 @@
 3. Select `import theme` and browse to where you cloned Catppuccin
 4. Select it
 
-## 🙋 FAQ (optional)
+## 🙋 FAQ
```

Comment:
How about this?

```suggestion
<!-- this section is optional -->
## 🙋 FAQ
```

Reply:
Ahhh README comments, good catch! I totally forgot about them. 
Looks perfect

Reply:
Also, seeing how 3/5 ports you mentioned are Ruby's, do me a favor and ping them on Discord with their favorite catchphrase.